app/views/dashboard/associated_users/update.js.coffee

```diff
@@ -18,7 +18,8 @@
 # INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
 # TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 <% if @errors.present? %>
-$("#modal_errors").html("<%= escape_javascript(render('shared/modal_errors', errors: @errors)) %>")
+$("#modal_place #modal_errors").html("<%= escape_javascript(render('shared/modal_errors', errors: @errors)) %>")
+$("#modal_place").scrollTop(0)
```

Comment:
We should probably also add this to app/views/dashboard/associated_users/create.js.coffee